app.rb

```diff
@@ -56,25 +57,27 @@ def get_param(key, acceptable_values)
     end
   end
 
-  post '/login_get/?' do
-    puts "Logging in via GET"
+  get '/login_get/?' do
```

Comment:
Changed this to be a GET to make it easier to redirect in the case of the step-up flow.